modules/admin_manual/pages/installation/quick_guides/ubuntu_22_04.adoc

```diff
@@ -0,0 +1,328 @@
+= Install ownCloud on Ubuntu 22.04
+:toc: right
+
+== Introduction
+
+This is a short guide to installing ownCloud on a fresh installation of Ubuntu 22.04. Run the following commands in your terminal to complete the installation.
+
+This guide can not go into details and has its limits by nature. If you experience issues like with dependencies of PHP or other relevant things like the operating system, web server or database, look at the xref:installation/manual_installation/manual_installation.adoc#ubuntu-22-04-lts-server[Detailed Installation Guide] for more information.
+
+== Prerequisites and Notes
+
+* A fresh installation of https://www.ubuntu.com/download/server[Ubuntu 22.04] with SSH enabled.
+* This guide assumes that you are working as the root user.
+* Your ownCloud directory will be located in `/var/www/owncloud/`.
+* php 7.4 from the ondrej/php PPA is going to be installed on Ubuntu 22.04.
+* Use the correct xref:installation/manual_installation/manual_installation_apache.adoc#multi-processing-module-mpm[Multi-Processing Module (MPM)].
+
+== Preparation
+
+=== Set Your Domain Name
+
+[source,bash]
+----
+my_domain="Your.Domain.tld"
+echo $my_domain
+
+hostnamectl set-hostname $my_domain
+hostname -f
+----
+
+=== Generate Strong Passwords
+
+----
+sec_admin_pwd=$(openssl rand -base64 18)
+echo $sec_admin_pwd > /etc/.sec_admin_pwd.txt
+
+sec_db_pwd=$(openssl rand -base64 18)
+echo $sec_db_pwd > /etc/.sec_db_pwd.txt
+----
+
+=== Update Your System
+
+First, ensure that all the installed packages are entirely up to date and that PHP is available in the APT repository.
+To do so, follow the instructions below:
+
+[source,bash]
+----
+apt update && \
+  apt upgrade -y
+----
+
+=== Create the occ Helper Script
+
+Create a helper script to simplify running xref:configuration/server/occ_command.adoc[occ commands]:
+
+[source,bash]
+----
+FILE="/usr/local/bin/occ"
+cat <<EOM >$FILE
+#! /bin/bash
+cd /var/www/owncloud
+sudo -E -u www-data /usr/bin/php /var/www/owncloud/occ "\$@"
+EOM
+----
+
+Make the helper script executable:
+
+[source,bash]
+----
+chmod +x $FILE
+----
+
+=== Install the Required Packages
+
+[source,bash]
+----
+sudo add-apt-repository ppa:ondrej/php -y
+sudo apt update && sudo apt upgrade
+
+apt install -y \
+  apache2 \
+  libapache2-mod-php7.4 \
+  mariadb-server \
+  openssl redis-server wget \
+  php7.4 php7.4-imagick php7.4-common \ 
+  php7.4-curl php7.4-gd php7.4-imap \
+  php7.4-intl php7.4-json php7.4-mbstring \
+  php7.4-mysql php7.4-ssh2 php7.4-xml \
+  php7.4-zip php7.4-apcu php7.4-redis \
+  php7.4-ldap php-opcache \
```

Comment:
It should be php7.4-opcache otherwise you will get an `Package 'php-opcache' has no installation candidate` error